### PR TITLE
Modify display and style of API tokens

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Slight modification to API secrets display

### CODE REVIEW

File: app/liquid_tags/slideshare_tag.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class SlideshareTag < LiquidTagBase
```
Comment (abraham) [inline app/liquid_tags/slideshare_tag.rb]: This looks like a mistaken inclusion.

**Comment by rhymes** [comment]
I have a few "odd" questions regarding the API that I'm not sure where to ask so I'll put them here :D

* is it documented somewhere?
* is it okay to publish it under `/v0` ?
* is it the same that's used by the client side JS? 

Thanks!

File: app/liquid_tags/slideshare_tag.rb (hunk 1 above)
Comment (benhalpern) [reply app/liquid_tags/slideshare_tag.rb]: Thanks! Good catch.

**Comment by benhalpern** [comment]
@rhymes 

> is it documented somewhere?

Not yet, but we'll have some coming very soon now that we have this.

> is it okay to publish it under /v0 ?

Can you clarify exactly what you're asking here?

> is it the same that's used by the client side JS?

Yes, but without a ton of consistency here. We currently don't have a true API but now that we have tokens, we'll start rolling out the features.

**Comment by rhymes** [comment]
@benhalpern awesome! Thanks for the questions

>> is it okay to publish it under /v0 ?
> Can you clarify exactly what you're asking here?

This was badly worded, my bad. You basically answered in the following answer

**Pull request merged**